src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs

```diff
@@ -156,7 +160,11 @@ public string GetBaggageItem(string key)
         public Activity(string operationName)
         {
             if (string.IsNullOrEmpty(operationName))
-                throw new ArgumentException($"{nameof(operationName)} must not be null or empty");
+            {
+                NotifyError(new ArgumentException($"{nameof(operationName)} must not be null or empty"));
```

Comment:
> If we are uniform we are about this in CoreFX in general, then it makes sense to log a issue to fix it here

We try to consistently use resources for error messages.  Places we don't are generally considered bugs to be fixed.